internal/resources/google/compute_cost_component_helpers.go

```diff
@@ -412,6 +412,9 @@ func guestAcceleratorCostComponent(region string, purchaseOption string, guestAc
 	case "nvidia-tesla-a100":
 		name = "NVIDIA Tesla A100"
 		descPrefix = "Nvidia Tesla A100 GPU"
+	case "nvidia-l4":
+		name = "NVIDIA L4"
+		descPrefix = "Nvidia L4 GPU"
```

Comment:
We had a discussion in planning about replacing this switch entirely as the `name` and `descPrefix` are just expanded and uppercased strings of the `guestAcceleratorType`. Can we change please.